controllers/apply/__tests__/Untitled-1

```diff
@@ -1,38 +1,39 @@
 /* eslint-env jest */
```

Comment:
Is this filename right?

Reply:
Um, nope that's not meant to be here at all!